cea/demand/thermal_loads.py

```diff
@@ -1186,6 +1186,12 @@ def get_temperatures(locator, prop_HVAC):
     type_cs            T0   (copied from input)
     type_dhw           T1   (copied from input)
     type_ctrl          T1   (copied from input)
+    type_vent          T1   (copied from input)
+    MECH_VENT        True   (copied from input, ventilation system configuration)
```

Comment:
I was wondering why you switched to uppercase for this.. care to explain? Also, I believe using the whole word would be easier to read and maintain: `mechanical_ventilation`, `window_ventilation`, `heat_recovery`, `night_f????`, `economizer`. What do you think?